.github/workflows/release.yml

```
@@ -0,0 +1,60 @@
+name: Tag Release
+
+on: 
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  build_linux:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: autogen
+        run: ./autogen.sh
+      - name: configure
+        run: ./configure
+      - name: make
+        run: make
+      - name: make check
+        run: make check
+      - name: make distcheck
+        run: make distcheck
+      - uses: actions/upload-artifact@v2
+        with:
+          name: linux64-qstat
```

```suggestion
          name: linux_amd64
```
Standardised naming that also supports bulk downloads